lib/LoginStrategy.ts

```diff
@@ -164,6 +182,32 @@ export class LoginStrategy extends BaseStrategy {
     }
   }
 
+  async loginToCacheServer() {
+    const [address, blockNumber] = await Promise.all([
+      this.signer.getAddress(),
+      this.provider.getBlockNumber(),
+    ])
+
+    const {
+      encodedHeader,
+      encodedPayload,
+    } = createLoginTokenHeadersAndPayload({ address, blockNumber })
+    const msg = `0x${Buffer.from(`${encodedHeader}.${encodedPayload}`).toString(
+      'hex'
+    )}`
+    const hash = keccak256(msg)
+    const sig = await this.signer.signMessage(arrayify(hash))
+    const encodedSignature = base64url(sig)
+    const claim = `${encodedHeader}.${encodedPayload}.${encodedSignature}`
+    const { data } = await this.httpClient.post<{ token: string }>('/login', {
+      claim,
+    })
+    this.httpClient.defaults.headers.common[
+      'Authorization'
+    ] = `Bearer ${data.token}`
+    console.log('DIDStrategy is now logged to cache server')
```

Comment:
Minor point but message might be a bit clearer if it said 'LoginStrategy is now logged in to cache server"